pkg/sql/conn_executor_savepoints.go

```diff
@@ -152,6 +152,12 @@ func (ex *connExecutor) execRelease(
 		// Committing the transaction failed. We'll go to state RestartWait if
 		// it's a retriable error, or to state RollbackWait otherwise.
 		if errIsRetriable(err) {
+			// For certain retryable errors, we should turn them into client visible
+			// errors, since the client needs to retry now.
+			var converted bool
```

Comment:
ooc, why not just use `:=` in the below if statement? Is there not a compiler optimization that renders these equivalent in performance or are we intentionally trying to leak out `err`?